File: SED-Front/src/app/modelo/comportamiento-plantilla.ts
export class ComportamientoPlantilla {
    id?: number;
    descComportamiento : string;
    grado : string;
    valoracionNumerica : number;
    esAprobada : boolean;

    constructor(descComportamiento:string,grado:string,valoracionNumerica:number,esAprobada:boolean) {
        this.descComportamiento = descComportamiento;
        this.grado = grado;
        this.valoracionNumerica = valoracionNumerica;
        this.esAprobada = esAprobada;
    }
}

File: SED-Back/src/main/java/com/sedback/SEDBack/Modelo/ComportamientoPlantilla.java
package com.sedback.SEDBack.Modelo;

import java.io.Serializable;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data //Agrega las caracteristicas seters y geters, ToString,etc
@NoArgsConstructor
public class ComportamientoPlantilla implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    
    private String descComportamiento;
    private String grado;
    private int valoracionNumerica;
    private boolean esAprobada;
}
